build-logic/settings-plugins/src/main/kotlin/com.hedera.hashgraph.settings.settings.gradle.kts

```diff
@@ -24,7 +24,6 @@ pluginManagement {
 
 plugins {
     id("com.gradle.enterprise")
-    id("org.gradle.toolchains.foojay-resolver-convention")
```

Comment:
We cannot use the auto-provisioning, there is no point in adding this plugin for it.